base_phone/models/phone_common.py

```diff
@@ -5,11 +5,15 @@
 from openerp import models, api
 from openerp.addons.base_phone.fields import Phone
```

Comment:
Ok, that suggestion seems to have worked in base_phone/models/res_partners.py but not in phone_common.py.

Partial backtrace:
  File "/usr/lib/python2.7/site-packages/openerp/modules/module.py", line 324, in load_openerp_module
    __import__('openerp.addons.' + module_name)
  File "/usr/lib/python2.7/site-packages/openerp/modules/module.py", line 61, in load_module
    mod = imp.load_module('openerp.addons.' + module_part, f, path, descr)
  File "/usr/lib/python2.7/site-packages/openerp/addons/base_phone/__init__.py", line 3, in <module>
    from . import fields